# Default initial positions for tidybot's ros2_control fake system

initial_positions:
<<<<<<< HEAD
=======
  joint_x: 0
  joint_y: 0
  joint_th: 0
>>>>>>> master
  joint_1: 0
  joint_2: 0
  joint_3: 0
  joint_4: 0
  joint_5: 0
  joint_6: 0
  right_finger_bottom_joint: 0